### PR TITLE
[doxygen] Add revision when API was changed or expanded

### PR DESCRIPTION
**Pull request by MartijnKaijser**
## Description

Expand the Doxygen PyDocs with the version the API was added/changed. Makes it easier for developer to know when things changed or were added.
Included most of the v17 changes and whatever i came across.

Example:
API change overview
https://codedocs.xyz/MartijnKaijser/xbmc/python_v17.html

Function overview with link added
https://codedocs.xyz/MartijnKaijser/xbmc/group__python___info_tag_video.html

### CODE REVIEW

File: xbmc/addons/kodi-addon-dev-kit/doxygen/Modules/modules_python.dox [hunk 1]
```diff
@@ -141,3 +141,27 @@ web applications or frameworks for the Python programming language.
 @}
 
 */
+/** @page python_v14 Python v14 API Changes
+ *  @brief Overview of changes done in Kodi v14 Python API
+ *
+ *  Changes page contents.
+ */
+
```
Comment (AlwinEsch) [inline xbmc/addons/kodi-addon-dev-kit/doxygen/Modules/modules_python.dox]: Here about a global to have all of them in one another Page together and not on top:

/** @page python_revisions Python API Changes
  *  @brief Overview of changes done Python API for Kodi
  *
  *  @subpage python_v14 
  *  @subpage python_v15
  *  @subpage python_v16
  *  @subpage python_v17
  */

**Comment by MartijnKaijser** [comment]
jenkins build and merge

**Comment by MartijnKaijser** [comment]
jenkins build and merge

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by MartijnKaijser** [comment]
jenkins build and merge

**Pull request merged**